package.json

```diff
@@ -71,7 +71,7 @@
     "@kie/mock-github": "^1.0.0",
     "@oguzhnatly/react-native-image-manipulator": "github:Expensify/react-native-image-manipulator#5cdae3d4455b03a04c57f50be3863e2fe6c92c52",
     "@onfido/react-native-sdk": "8.3.0",
-    "@react-native-async-storage/async-storage": "^1.19.5",
+    "@react-native-async-storage/async-storage": "1.21.0",
```

Comment:
what's this change for?

Reply:
It was required, because of a native dependency called `room` in HybridApp